ui/static/ui/js/learning_resources.jsx

```diff
@@ -476,19 +529,13 @@ define('learning_resources',
     TermSelect: TermSelect,
     VocabSelect: VocabSelect,
     LearningResourcePanel: LearningResourcePanel,
-    loader: function (
-      repoSlug, learningResourceId, refreshFromAPI,
-      closeLearningResourcePanel, container
-    ) {
-      // Unmount and remount the component to ensure that its state
-      // is always up to date with the rest of the app.
-      React.unmountComponentAtNode(container);
-      React.render(<LearningResourcePanel
-        repoSlug={repoSlug}
-        learningResourceId={learningResourceId}
-        refreshFromAPI={refreshFromAPI}
-        closeLearningResourcePanel={closeLearningResourcePanel}
-        />, container);
+    loader: function (options, isNewResource, container) {
+      if (isNewResource) {
```

Comment:
We don't need to know whether the resource is new or not, just whether it's dirty or not. I'm fine with the `options` change but we should remove `isNewResource` unless it's necessary for this task. If it's there only to save an API call, that should go in a separate PR